rundeckapp/grails-app/services/rundeck/services/AuthStorageManager.groovy

```diff
@@ -0,0 +1,27 @@
+package rundeck.services
+
+import com.dtolabs.rundeck.core.authorization.AuthContext
+import com.dtolabs.rundeck.core.storage.ResourceMeta
+import com.dtolabs.rundeck.core.storage.StorageTree
+import com.dtolabs.rundeck.core.storage.StorageUtil
+import com.dtolabs.rundeck.core.storage.AuthStorageManager
+import grails.gorm.transactions.Transactional
+import org.rundeck.storage.data.MemoryTree
+
+@Transactional
+class AuthStorageTreeService implements AuthStorageManager{
```

Comment:
should avoid adding Service classes to the grails services, which don't need to be grails services.  This class could be removed, and simply inject the `rundeckAuthStorageManager` bean defined here https://github.com/rundeck/rundeck/pull/5392/files#diff-52259838c7d67dfe63914241dcf89443R114

Reply:
i think...